js/main.js

```diff
@@ -0,0 +1,15 @@
+'use strict'
+
+const getRandomInt = function (min, max) {
+  return Math.floor(Math.random() * (max - min + 1)) + min;
+};
+
+const  checkMaxStringValue = function (str, max)  {
+  return str.length <= max ? true : false;
```

Comment:
Условие и так возвращает булевое значение, так что отдельно указывать `true` и `false` незачем 
```suggestion
  return str.length <= max;
```